sqlmesh/core/renderer.py

```diff
@@ -310,10 +314,12 @@ def __init__(
         macro_definitions: t.List[d.MacroDef],
         schema: t.Optional[t.Dict[str, t.Any]] = None,
         model_name: t.Optional[str] = None,
+        model_fqn: t.Optional[str] = None,
         path: Path = Path(),
         jinja_macro_registry: t.Optional[JinjaMacroRegistry] = None,
         python_env: t.Optional[t.Dict[str, Executable]] = None,
         only_execution_time: bool = False,
+        default_catalog: t.Optional[str] = None,
```

Comment:
Is this `default_catalog` different from the one pass in constructor? I'm somewhat confused here.

Reply:
I'm not sure what is confusing here. We are overriding the paren't constructor and the parent needs `default_catalog` so we need `default_catalog`. Is it the order of the fields that adds confusion?

Reply:
Nope, that's my bad, sorry. I've misinterpreted what github presented to me and thought this was related to the `render` method definition and not the definition of the constructor.